guillotina/contrib/catalog/pg.py

```diff
@@ -581,15 +583,28 @@ def load_meatdata(self, query: ParsedQueryInfo, data: typing.Dict[str, typing.An
         results = []
         try:
             context_url = get_object_url(container)
+            request = get_current_request()
+            fullobjects = request.query.get('_fullobjects', 'false').lower() != 'false'
```

Comment:
`_fullobjects` should be added to the query spec instead of looking at query param directly here.

here https://github.com/plone/guillotina/blob/master/guillotina/catalog/types.py#L6
or here https://github.com/plone/guillotina/blob/master/guillotina/contrib/catalog/pg.py#L71

Reply:
Agreed, not willing to change the contract of the search endpoint so we need to update also ES